admin/win/Toolchain-mingw32-openSUSE.cmake

```diff
@@ -27,8 +27,8 @@ SET(QT_MKSPECS_DIR          ${CMAKE_FIND_ROOT_PATH}/share/qt5/mkspecs)
 SET(QT_QT_INCLUDE_DIR       ${CMAKE_FIND_ROOT_PATH}/include)
 
 # qt tools
-SET(QT_QMAKE_EXECUTABLE     ${MINGW_PREFIX}-qmake )
-SET(QT_MOC_EXECUTABLE       ${MINGW_PREFIX}-moc)
-SET(QT_RCC_EXECUTABLE       ${MINGW_PREFIX}-rcc)
-SET(QT_UIC_EXECUTABLE       ${MINGW_PREFIX}-uic)
-SET(QT_LRELEASE_EXECUTABLE  ${MINGW_PREFIX}-lrelease)
+SET(QT_QMAKE_EXECUTABLE         ${MINGW_PREFIX}-qmake-qt5)
+SET(QT_MOC_EXECUTABLE           ${MINGW_PREFIX}-moc-qt5)
+SET(QT_RCC_EXECUTABLE           ${MINGW_PREFIX}-rcc-qt5)
+SET(Qt5Widgets_UIC_EXECUTABLE   ${MINGW_PREFIX}-uic-qt5)
```

Comment:
I don't understand this change.
Why is it needed and why not for qmake, moc,rcc,lrelease ?